code/modules/mob/living/carbon/alien/larva/larva.dm

```diff
@@ -5,8 +5,8 @@
 	pass_flags = PASSTABLE | PASSMOB
 	mob_size = MOB_SIZE_SMALL
 
-	maxHealth = 30
-	health = 30
+	maxHealth = 25
+	health = 25
```

Comment:
So, explanation for this. This was always meant to be 25--and I adjusted it before, but when we used the older crit system, setting it to 30 made them....literally invincible.

Reply:
Not 100% sure, I think it was something to do with how `update_stat` was written back in the day and not being able to hit appropriately negative values.